.github/workflows/codecov.yml

```diff
@@ -33,20 +32,27 @@ jobs:
     strategy:
       fail-fast: true
     steps:
-      - name: install toolchain according to rust-toolchain
-        uses: actions-rs/toolchain@v1
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
         with:
-          toolchain: nightly
+          toolchain: beta
           profile: default
           override: true
           components: llvm-tools-preview
-      - name: Checkout sources
-        uses: actions/checkout@v2
-      - uses: Swatinem/rust-cache@v1
+      # Enable caching of the 'librocksdb-sys' crate by additionally caching the
+      # 'librocksdb-sys' src directory which is managed by cargo
+      - uses: bmwill/rust-cache@v1 # Fork of 'Swatinem/rust-cache' which allows caching additional paths
+        with:
+          path: ~/.cargo/registry/src/**/librocksdb-sys-*
       - name: Install grcov, and cache the binary
         uses: baptiste0928/cargo-install@v1
         with:
           crate: grcov
+          locked: true
+      # Normally generating code coverage requires use of a nightly compiler
+      # In order to have a more stable and less noisy experience, lets instead
+      # opt to use the stable toolchain specified via the 'rust-toolchain' file
+      # and instead enable nightly features via 'RUSTC_BOOTSTRAP'
```

Comment:
👍

Reply:
oops that comment is out dated lol. I actually tried this and it failed...because the feature gets stabilized in 1.60 (which is just a few short days away) and the `-C` variant doesn't exist in 1.59 so I opted to use beta for a couple days and when it stabilizes we can just use stable going forward.